sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -664,27 +664,34 @@ protected virtual BlobBaseClient WithSnapshotImpl(string snapshot)
         /// notifications that the operation should be cancelled.
         /// </param>
         /// <returns>
-        /// A <see cref="Response{BlobCopyInfo}"/> describing the
+        /// A <see cref="CopyFromUriOperation"/> describing the
         /// state of the copy operation.
         /// </returns>
         /// <remarks>
         /// A <see cref="StorageRequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
-        public virtual Response<BlobCopyInfo> StartCopyFromUri(
+        public virtual CopyFromUriOperation StartCopyFromUri(
```

Comment:
After preview we should try to return Operation<T>

Reply:
Agreed - once we add `Operation<T>.Id`.